<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrar</class>
 <widget class="QDialog" name="borrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Archivo(con registros)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboarchivos_borrar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combocampos_borrar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Escoja el campo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Busqueda:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lbusqueda_borrar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_borrar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboregistros_borrar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar registro para agregar a la lista de borrar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar lista</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar todos los registros</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar registros seleccionados</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
